README.md

```diff
@@ -68,13 +83,16 @@ We follow [semantic versioning](https://semver.org/) and are guided by the princ
 We may occasionally make breaking changes in order to improve the user experience.
 When possible, we will keep old interfaces and mark them as deprecated, as long as they can co-exist with the
 new ones.
-Each substantial improvement, breaking change, or deprecation will be documented in the release notes.
+Each substantial improvement, breaking change, or deprecation will be documented in the
+[release notes](https://qiskit.github.io/qiskit-addon-sqd/release-notes.html).
 
 ----------------------------------------------------------------------------------------------------
 
 ### Contributing
 
-The developer guide is located at [CONTRIBUTING.md](https://github.com/Qiskit/qiskit-addon-sqd/blob/main/CONTRIBUTING.md>)
+The source code is available [on GitHub](https://github.com/Qiskit/qiskit-addon-sqd).
```

Comment:
I took this out of README because I think you are on GitHub if you are reading the README generally, but I guess that's a bit of a stretch assumption